docs/openapi_v1.0.0-alpha.yaml

```
@@ -202,18 +211,19 @@ paths:
         '500':
           $ref: '#/components/responses/InternalServerError'
 
-  /reservations:
-    post:
+  /suppliers/{supplierId}/bookings:
```

This endpoint definition is missing a placeholder for the `uuid` path parameter called out in the definition below.

```suggestion
  /suppliers/{supplierId}/bookings/{uuid}:
```